README.md

```diff
@@ -195,3 +200,13 @@ although it is of note that you can find these files directly where you your `fi
 [codecov-link]: https://codecov.io/gh/aiidateam/aiida-firecrest
 [black-badge]: https://img.shields.io/badge/code%20style-black-000000.svg
 [black-link]: https://github.com/ambv/black
+
+
+
+#### Mocking PyFirecREST
+
+These set of test do not gurantee that the firecrest protocol is working, but it's very usefull to quickly check if `aiida-firecrest` is behaving as it's expected to do. To run just simply use `pytest`.
+
+
+If these tests, pass and still you have trouble in real deploymeny that means your installed version of pyfirecrest is behaving differently from what `aiida-firecrest` expects in `MockFirecrest` in `tests/tests_mocking_pyfirecrest/conftest.py`.
+In order to solve that, first spot what is different and then solve or raise to `pyfirecrest` if relevant.
```

Comment:
```suggestion
If there is no version of `aiida-firecrest` available that supports your `pyfirecrest` version and if down/upgrading your `pyfirecrest` to a supported version is not an option, you might try the following:
- open an issue on the `aiida-firecrest` repository on GitHub to request supporting your version of pyfirecrest
- if you feel up to finding the discrepancy and fixing it within `aiida-firecrest`, open a PR instead
- if you think the problem is a bug in `pyfirecrest`, open an issue there

Either way, make sure to report which version of `aiida-firecrest` and `pyfirecrest` you are using.
```

I feel like there should be an option here for people who don't have the know-how to fix the problem themselves or even figure out where the problem lies, beyond mismatching versions.